### PR TITLE
work around timeout for first test to get run taking too long

### PR DESCRIPTION
**Pull request by paulcallen**
Signed-off-by: Paul Allen <paul.c.allen@microsoft.com>

### CODE REVIEW

File: tests/fork/Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ tests:
 
 # no fork
 test-no-fork: rootfs
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) --app-config-path config-none.json rootfs /bin/fork nofork
+	TIMEOUT=360 $(RUNTEST) $(MYST_EXEC) $(OPTS) --app-config-path config-none.json rootfs /bin/fork nofork
```
Comment (jxyang) [inline tests/fork/Makefile:30]: I'd love to understand why this simple test could take 6 minutes.

Comment (paulcallen) [reply tests/fork/Makefile:30]: Basically _something_ is causing the first test to get executed to take an undetermined amount of time. I have not been able to narrow this down more at this point, but my current thoughts are that one of the shared libraries the myst tool links again is doing something during the initialization of the library and that sometimes takes longer than it should.
It generally takes less than a couple of minutes, but sometimes it is a little more.
The hang happens _before_ main() is even hit in the process which is why I think this is something to do with the first test being called.

File: tests/myst/mkcpio/Makefile [hunk 2]
```diff
@@ -16,7 +16,7 @@ REDEFINE_TESTS=1
 include $(TOP)/rules.mak
 
 tests:
-	$(RUNTEST) $(MAKE) test-mkcpio
```
Comment (vtikoo) [inline tests/myst/mkcpio/Makefile:19]: Is this intentional?

Comment (paulcallen) [reply tests/myst/mkcpio/Makefile:19]: yep! There are several things in the individual test that take a long time and each has $(runtest). Doing it at this layer can cause the test to timeout prematurely. Creating large cpio/ext2 images can take a bit of time.

**Review by vtikoo — APPROVED** [review]

File: tests/fork/Makefile (hunk 1 above)
Comment (jxyang) [reply tests/fork/Makefile:30]: I am not a fan of making tests pass by increasing the time outs or memory sizes. The actual question we should ask is: how could a previous version of Mystikos pass the test with a smaller time out or memory size. If we continue on the path of increasing time outs or memory sizes, we are on a deteriorating path.

Comment (vtikoo) [reply tests/fork/Makefile:30]: We should definitely rootcause this, but there is also the issue of this blocking PRs.

**Pull request merged**